components/Venue/venue.js

```diff
@@ -1,26 +1,38 @@
 /* eslint-disable react/no-unescaped-entities */
 import React from 'react';
 
+ function isEventEnded(date){
```

Comment:
It would be great if you could export the function from Line 4. no need to export explicitly in line 44 `export function isEventEnded(){}`

Reply:
Done